packages/api-providers/src/abstracts/relayer/types.ts

```diff
@@ -208,7 +208,7 @@ export type MixerRelayTx = {
  * if it passed as zero nothing will happen unless a real value is passed thus a new note isn't generated
  * @param roots - roots bytes array
  **/
-type AnchorRelayTransaction = {
+type AnchorRelayedTransaction = {
```

Comment:
Revert naming to be consistent with the added `VAnchorRelayTransaction` type, as well as it should be present tense as it represents a transaction that is sent to the relayer and has not yet been relayed.